roles/tpa_single_node/templates/manifests/bombastic/walker/Deployment.yaml.j2

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bombastic-walker
+  namespace: "{{ tpa_single_node_namespace }}"
+  labels:
+    app.kubernetes.io/name: bombastic-walker
+    app.kubernetes.io/component: walker
+    app.kubernetes.io/part-of: trustification
+spec: 
+  selector:
+    matchLabels:
+      app: vexination-walker
+  template:
+    spec:
+      restartPolicy: Never
+      volumes:
+        - name: walker-state
+          persistentVolumeClaim:
+            claimName: bombastic-walker-state
+      containers:  
+        - image: "{{ tpa_single_node_trustification_image }}"
+          imagePullPolicy: Always
+          name: walker
+          command:
+            - /trust
+          args:
+            - bombastic
+            - walker
+            - --sink
+            - http://127.0.0.1:{{ tpa_single_node_bombastic_api_port }}/api/v1/sbom
```

Comment:
I hope it should be http://bombastic-api-pod:{{ tpa_single_node_bombastic_api_port }}/api/v1/sbom as the walker needs to reach out bombastic api pod and the localhost represent the bombastic walker pod.

Reply:
@mrrajan, you're right, podman behavior is different than k8s cluster, thanks for catching that !